### ISSUE TITLE
[Stats] Gather data for Competition Statistics

### ISSUE THREAD

**Issue by w1lla** [skybaks/pyplanet-cup_manager#44]
Hi,

Make a widget/window to gather data for instance that who hosted the last cup but also who hosted the most cups, won the most cups, who always ended as last in a certain cup name?

**Comment by skybaks** [comment]
Yeah now that the plugin is tracking competitions and we have that info in the database we could make this view. I'll have to think about the implementation that makes the most sense to users and admins
